compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -599,8 +597,7 @@ trait ParallelTesting extends RunnerOrchestration { self =>
       val command = scalacCommand ++ flags1.all ++ files.map(_.getAbsolutePath)
       val process = Runtime.getRuntime.exec(command)
 
-      val reporter = TestReporter.reporter(realStdout, logLevel =
-        if (suppressErrors || suppressAllOutput) ERROR + 1 else ERROR)
+      val reporter = mkReporter
```

Comment:
thanks, I touched this odd syntax in my PR for skipping tests, I will revert that; and also get that PR over the line.